api/types/session_tracker.go

```diff
@@ -252,7 +252,7 @@ func (s *SessionTrackerV1) AddParticipant(participant Participant) {
 func (s *SessionTrackerV1) RemoveParticipant(id string) error {
 	for i, participant := range s.Spec.Participants {
 		if participant.ID == id {
-			s.Spec.Participants = append(s.Spec.Participants[:i], s.Spec.Participants[i+1:]...)
+			s.Spec.Participants[i], s.Spec.Participants = s.Spec.Participants[len(s.Spec.Participants)-1], s.Spec.Participants[:len(s.Spec.Participants)-1]
```

Comment:
Maybe a non-issue but I think this splicing method will change element ordering: https://go.dev/play/p/4gt5jIWzCcV

And subjectively at least I had to read it a number of times to figure out what it was trying to do, if we're okay with the different ordering maybe we can just split this into two lines to help readability a bit?

```go
s.Spec.Participants[i] = s.Spec.Participants[len(s.Spec.Participants)-1]
s.Spec.Participants = s.Spec.Participants[:len(s.Spec.Participants)-1]
```

Reply:
No worries, it's nothing important, I was just trying to understand the intent.